### PR TITLE
French.md

### PR DESCRIPTION
**Pull request by Daksheshapkare**
Solving Issue 
https://github.com/Ashish-khanagwal/Open-source-practice-and-resources/issues/94

### CODE REVIEW

**Comment by Daksheshapkare** [comment]
Please take a look at this 
And let me know if any changes

File: Translation of readme in French.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# :wave: Passerelle vers l'Open Source | Meilleures ressources de programmation
+<p align="center"><img src="https://user-images.githubusercontent.com/75534912/194343020-6cbd2485-c2ea-4779-87e2-9708238398e7.png"></p>
+
+# Gateway to Open Source | Best Programming Resources
+## Agenda:rocket:
+L'objectif de la communauté à partir de ce référentiel n'est pas seulement de fournir les meilleures ressources/contenus, mais aussi de motiver/encourager les débutants qui n'ont jamais contribué à l'Open Source, et de les aider à démarrer avec l'Open Source de manière simple avec des projets amicaux et avec un support complet
```
Comment (Ashish-khanagwal) [inline Translation of readme in French.md:6]: Translate both lines `4` and `5` into French too

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]
Rename this file from `Translation of readme in French.md` to `French.md`

**Comment by Daksheshapkare** [comment]
> Rename this file from `Translation of readme in French.md` to `French.md`

Done

**Comment by Daksheshapkare** [comment]
Please check this and merge it

File: Translation of readme in French.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # :wave: Passerelle vers l'Open Source | Meilleures ressources de programmation
 <p align="center"><img src="https://user-images.githubusercontent.com/75534912/194343020-6cbd2485-c2ea-4779-87e2-9708238398e7.png"></p>
 
-# Gateway to Open Source | Best Programming Resources
-## Agenda:rocket:
+# Passerelle vers l'Open Source | Meilleures ressources de programmation
+
```
Comment (Ashish-khanagwal) [inline Translation of readme in French.md:5]: remove this extra line `5`

File: Translation of readme in French.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# :wave: Passerelle vers l'Open Source | Meilleures ressources de programmation
```
Comment (Ashish-khanagwal) [inline Translation of readme in French.md:1]: Please, This is not as Code For Community welcomes you
Have a look at the original repo's heading before the banner.
I can't help you any more with it.

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

File: Translation of readme in French.md (hunk 2 above)
Comment (Daksheshapkare) [reply Translation of readme in French.md]: removed line 5

File: Translation of readme in French.md (hunk 1 above)
Comment (Daksheshapkare) [reply Translation of readme in French.md:7]: translatated

**Comment by Daksheshapkare** [comment]
@Ashish-khanagwal 
done with your changes

**Review by Ashish-khanagwal — APPROVED** [review]
Thank you for your contribution :rocket:

**Pull request merged**